### ISSUE TITLE
How to define accept-language in request header?

### ISSUE THREAD

**Issue by fishjar** [swaggo/swag#1289]
**Is your feature request related to a problem? Please describe.**
I'm not sure if there is a feature for this, does anyone know how to do it?

**Comment by ubogdan** [comment]
This should work, set false to true if this header is required
```go
// @Param   accept-language  header     string     false  "some description"    

```

**Comment by fishjar** [comment]
> This should work, set false to true if this header is required
> 
> ```go
> // @Param   accept-language  header     string     false  "some description"    
> ```

Yes it works, just need to be set repeatedly for each handler, It would be better if it could be set globally.
